### PR TITLE
Psg 4981 use repo templates

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
## What's New?

Uses repo templates to format README, LICENSE, and remove community files in favour of using organization community files.

Removed bulk of the old readme as it is all contained in our docs!

## Screenshots (if appropriate):

CleanShot 2024-10-22 at 16 16 00@2x

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have manually tested my code thoroughly
- [x] I have added/updated inline documentation for public facing interfaces if relevant
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing integration and unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update documentation and use org-level community files in repository",
```
Comment (vanessa-passage) [inline change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json:3]: Curiosity: How do the community files get updated? Do we have to explicitly do this?

**Review by vanessa-passage — APPROVED** [review]
Overall looks good. Just a question about how files are inherited, not a blocker.

File: change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json (hunk 1 above)
Comment (flanagankp) [reply change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json:3]: Community files are in the special org level .github repo: https://github.com/passageidentity/.github

Comment (jenn-macfarlane) [reply change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json:3]: Ha, beat me to it!

Comment (vanessa-passage) [reply change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json:3]: Cool. I didn't know if there was somehting we had to also do in the repos. Thanks!

Comment (jenn-macfarlane) [reply change/@passageidentity-passage-flex-node-1ce21785-66a4-44fb-840b-06352841a50f.json:3]: Nope, that's the beauty of it!!

**Pull request merged**